spec/system/empoyee_registers_a_shipment_time_setting_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+describe 'Shipment time setting' do
+  let(:employee) { create(:employee) }
+  let(:employee_with_shipping_co) { create(:employee, :with_shipping_co) }
+  context 'when logged in with a valid employee' do
+    it 'register successfully' do
+      login_as employee_with_shipping_co, scope: :employee
+      visit root_path
+      click_link 'Configurar Prazo de Entrega'
+      click_link 'Adicionar Novo Prazo de Entrega'
+      fill_in 'Distancia Minima', with: '0'
+      fill_in 'Distancia Maxima', with: '100'
+      fill_in 'Prazo de Entrega', with: '2'
+      click_on 'Salvar'
+
+      expect(page).to have_content 'Configurações salvas com sucesso!'
+      expect(page).to have_content 'Intervalo de distância: 0 a 100KM'
+      expect(page).to have_content 'Prazo de Entrega: 2 dias úteis'
+    end
+
+    it 'fail to register with blank fields' do
+      login_as employee_with_shipping_co, scope: :employee
+      visit root_path
+      click_link 'Configurar Prazo de Entrega'
+      click_link 'Adicionar Novo Prazo de Entrega'
+      fill_in 'Distancia Minima', with: ''
+      fill_in 'Distancia Maxima', with: ''
+      fill_in 'Prazo de Entrega', with: ''
+      click_on 'Salvar'
+
+      expect(page).to have_content('não pode ficar em branco')
+      expect(page).to have_content('não pode ficar em branco')
+      expect(page).to have_content('não pode ficar em branco')
```

Comment:
```suggestion
      expect(page).to have_content('não pode ficar em branco', count: 3)
```

Ter três except não verificaria se tem na tela 3 vezes 😢